src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php

```diff
@@ -264,6 +219,9 @@ public function escapeValue($value)
         return $value;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function unescapeValue($value)
```

Comment:
@nicolas-grekas I added BC layers in the latest commits to allow but I'm not sure we should do this in this PR (this makes a lot of changes). Should I open a new PR?